sympy/geometry/line.py

```diff
@@ -358,12 +398,14 @@ def direction(self):
         Returns
         =======
 
-        p : a Point; the ray from the origin to this point is the
-            direction of `self`
+        Point :
+            the ray from the origin to this point is the
+            direction of `self`.
```

Comment:
There can be a better description than this. I don't think "the direction of `self`" is any useful